src/core/lifetime.d

```diff
@@ -2648,16 +2648,19 @@ if (!Init.length ||
 }
 
 /**
- * Allocate an exception of type `T` from the exception pool and call its constructor.
- * It has the same interface as `rt.lifetime._d_newclass()`.
- * `T` must be Throwable or derived from it, must declare an explicit ctor
- * and cannot be a COM or C++ class.
+ * Allocate an exception of type `T` from the exception pool.
+ * `T` must be `Throwable` or derived from it and cannot be a COM or C++ class.
+ *
+ * Bugs:
```

Comment:
This is not necessarily a bug since the compiler inserts the call to the constructor. Maybe you can replace it with "Note"?